api/src/wfl/service/rawls.clj

```diff
@@ -37,14 +39,62 @@
    (create-snapshot-reference workspace snapshot-id name "")))
```

Comment:
Does this create an empty description? (-Do we care?-)

Reply:
It does, it's optional all the way to Rawls but figured the support may help in manual testing.